test/e2e/spworkflow/e2e_test.sh

```diff
@@ -7,8 +7,7 @@ workspace=${GITHUB_WORKSPACE}
 GREENFIELD_REPO_TAG="v0.2.3-alpha.6"
 #GREENFIELD_CMD_TAG="feat-adaptor-sp-exit"
 GREENFIELD_CMD_TAG="a573d064056c82a5062430fbb87e32251bfe6d2b"
-# feat-sp-exit-refactor-sp-info
-GREENFIELD_GO_SDK_TAG="e9fcd4155668718b117f50ff3c5085c0a1227421"
+GREENFIELD_GO_SDK_TAG="develop"
```

Comment:
Once the branch changes, the action may not pass. It is best to use the commit id.

Reply:
fixed